chart/templates/operator.yaml

```diff
@@ -646,6 +646,18 @@ spec:
           capabilities:
             drop:
             - ALL
+      initContainers:
+      - args:
+        - |
+          while [[ $(kubectl get pods postgres-postgresql-0 -n kubeflex-system -o 'jsonpath={..status.conditions[?(@.type=="Ready")].status}') != "True" ]]; do
```

Comment:
What happens if the nested command `kubectl get pods postgres-postgresql-0 -n kubeflex-system -o 'jsonpath={..status.conditions[?(@.type=="Ready")].status}'` fails, in the sense of exiting with a non-zero status? Will that make the `[[ ... ]]` exit with a non-zero status?

I think that the following is more clearly safe.

```
while ! [[ $(kubectl get pods postgres-postgresql-0 -n kubeflex-system -o 'jsonpath={..status.conditions[?(@.type=="Ready")].status}') == "True" ]]; do
```